src/common/internal-plugins.js

```diff
@@ -131,11 +124,6 @@ module.exports = [
         return eval("require")("../language-markdown/parser-markdown").parsers
           .remark;
       },
-      // TODO: Delete this in 2.0
-      get markdown() {
```

Comment:
can't find history, why this removed?

Reply:
I guess the original idea was to call the parser `remark` instead of `markdown`, like we have `babel`, not `javascript`. Not sure we still want to go that way.

Reply:
I guess probably we remove this after we support `language` option